webots_ros2_control/include/webots_ros2_control/Ros2Control.hpp

```diff
@@ -20,12 +20,15 @@
 #include <vector>
 #include <thread>
 
-#include "hardware_interface/base_interface.hpp"
+#if FOXY || (ROLLING && MAIN)
```

Comment:
I am assuming that `rolling` and `galactic` have the same breaking changes. So `foxy` would be the exception?

Reply:
Yes exactly!